react/hooks/useFetchMore.js

```diff
@@ -199,7 +216,13 @@ export const useFetchMore = props => {
     const from = max(0, to - maxItemsPerPage + 1)
     setInfiniteScrollError(false)
     pageDispatch({ type: 'PREVIOUS_PAGE', args: { from } })
-    setQuery({ page: pageState.previousPage }, { replace: true, merge: true })
+
+    const queries = { page: pageState.previousPage }
+    if (fuzzy) queries.fuzzy = fuzzy
```

Comment:
you should abstract this logic in a function, its used like 5 times.

Reply:
I've just realized that I can accomplish the same result by using `fuzzy: fuzzy || fuzzy`.
I think it is better now.